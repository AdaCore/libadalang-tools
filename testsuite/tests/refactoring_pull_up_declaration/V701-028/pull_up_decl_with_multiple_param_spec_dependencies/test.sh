pull_up_declaration -P default.gpr --source main.adb --line 6 --column 17
