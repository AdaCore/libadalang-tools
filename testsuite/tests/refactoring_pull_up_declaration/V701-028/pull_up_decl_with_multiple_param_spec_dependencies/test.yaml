title: pull_up_decl_with_multiple_param_spec_dependencies

description: pull_up_declaration tool test

driver: shell_script
