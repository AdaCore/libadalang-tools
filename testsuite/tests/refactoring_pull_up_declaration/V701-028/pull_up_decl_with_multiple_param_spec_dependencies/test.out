main.adb
4:1-4:1 
   procedure Extracted (A : Integer;
                        B : Integer);

   procedure Extracted (A : Integer;
                        B : Integer) is
   begin
      Put_Line (A'Image);
      Put_Line (B'Image);
   end Extracted;

6:1-13:1 
15:16-15:16  (A => A,
                  B => B)

